packages/gatsby-transformer-raml/src/create-api-node.mjs

```diff
@@ -1,6 +1,5 @@
-import doRecursion  from "./utils/api/do-recursion.mjs";
-import parametersToArray  from "./utils/parameters-to-array.mjs";
-
+import doRecursion from './utils/api/do-recursion.mjs';
```

Comment:
Just a question. Why do we use mjs files? We should be safe to use .ts/.js to use modern import export syntax

Reply:
Also an old convention that we have for our utils in the raml transformer package afaik. But in this case it probably still makes sense to use modules.

Reply:
I think it's because we use ecmascript modules here, and probably the project default is to use commonjs modules. Using .mjs extension ensures node loads the files as ecmascript modules rather than commonjs modules